staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -3623,7 +3624,7 @@ func (d *NodeDescriber) Describe(namespace, name string, describerSettings Descr
 	var events *corev1.EventList
 	if describerSettings.ShowEvents {
 		if ref, err := reference.GetReference(scheme.Scheme, node); err != nil {
-			klog.Errorf("Unable to construct reference to '%#v': %v", node, err)
+			klog.ErrorS(err, "Unable to construct reference to", node)
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.

Reply:
Done